### ISSUE TITLE
when trying to configure jenkins it says 'not found'

### ISSUE THREAD

**Issue by oshai** [AdamNowotny/BuildReactor#38]
I am trying to configure jenkins builds to display, but there is an error. looks like something got messed up with the views - note the primaryView in the url:

NotFoundError: Not Found
http://[my-jenkins-host]/view/All/primaryView/api/json?tree=jobs[name]

**Comment by AdamNowotny** [comment]
Unless something changed in Jenkins that seems like correct url.
First it gets
```
http://<host>/api/json?tree=jobs[name,buildable],primaryView[name],views[name,url]
```
then gets details of each view from
```
http://<host>/view/[view_name]/api/json?tree=jobs[name]
```
In case of primary view the ```view_name``` is replaced with ```[view_name]/primaryView```.

What's the url that works in your case?

**Comment by oshai** [comment]
this works for me (without primaryView):  
http://<host>/view/All/api/json?tree=jobs[name]
my jenkins version is 1.597 which is not so new

**Comment by oshai** [comment]
it is still not working for me. is it published?

**Comment by AdamNowotny** [comment]
No, it's assigned to https://github.com/AdamNowotny/BuildReactor/milestones/0.11.0

Reopening for now. Will close once it's published

**Comment by oshai** [comment]
it is not working yet....
NotFoundError: Not Found
http://<host>/view/Amplify%2520View/api/json?tree=jobs%5Bname%5D
<http://jenk-10001-prod-nydc1.nydc1.outbrain.com/view/Amplify%2520View/api/json?tree=jobs%5Bname%5D>

the problem is that Amplify%2520View
<http://jenk-10001-prod-nydc1.nydc1.outbrain.com/view/Amplify%2520View/api/json?tree=jobs%5Bname%5D>
should
be Amplify%20View
<http://jenk-10001-prod-nydc1.nydc1.outbrain.com/view/Amplify%2520View/api/json?tree=jobs%5Bname%5D>
do
you know why it happens?

**Comment by AdamNowotny** [comment]
Overdid it, there were some 304 redirections when url had spaces or other special characters so wanted to decrease the number of calls and encode the url. It broke some other calls apparently...
Just published fix 0.11.1 so hope this will be fixed within an hour or so.
Thanks for letting me know so quickly!

**Comment by oshai** [comment]
now it works, thanks!
